src/analysis/flux_variability_analysis.jl

```diff
@@ -145,6 +145,10 @@ function _FVA_optimize_reaction(model, rid, ret)
     var = all_variables(model)[abs(rid)]
 
     @objective(model, sense, var)
+    if termination_status(model) != MOI.OPTIMIZE_NOT_CALLED &&
+       solver_name(model) in ["Gurobi"]
```

Comment:
- is this really gurobi-only?
- wouldn't it be better for the warm start to come from the previous fba?

Reply:
I am not sure which solvers support this and which don't. Previously JuMP indicated that this happens automatically. I think doing it this was ensures that the previous solution is used? The solution is "contained" in `model` right?

Reply:
yeah, still technically there's probably no reason to make this work
- feasible solution is usually found very quickly and can be copied from the previous problem (if we want to optimize that, we should optimize JuMP/solvers, not COBREXA)
- caching the previous states won't help much because they are usually on a completely different side of the polytope

Reply:
yeah, I agree! closing this PR ⚰️